pkg/network/address.go

```diff
@@ -73,6 +83,13 @@ func multiaddrStringFromHostAddr(host string) (string, error) {
 		return "", err
 	}
 
+	// Empty address in host :8080` generates `/dns4//tcp/8080` multiaddr
```

Comment:
if i understand comment correctly, there is missing opening backtick before `:8080`